src/towncrier/test/test_project.py

```diff
@@ -94,7 +103,8 @@ def test_missing_version(self):
             get_version(tmp_dir, "missing")
```

Comment:
Minor comment.

Maybe this is a better name 

```suggestion
            get_version(tmp_dir, "non-existing-subdir")
```